opm/simulators/wells/MSWellHelpers.hpp

```diff
@@ -64,7 +64,7 @@ namespace mswellhelpers
         for (size_t i_block = 0; i_block < y.size(); ++i_block) {
             for (size_t i_elem = 0; i_elem < y[i_block].size(); ++i_elem) {
                 if (std::isinf(y[i_block][i_elem]) || std::isnan(y[i_block][i_elem]) ) {
-                    OPM_THROW(Opm::NumericalIssue, "nan or inf value found after UMFPack solve due to singular matrix");
+                    OPM_THROW_NOLOG(Opm::NumericalIssue, "nan or inf value found after UMFPack solve due to singular matrix");
```

Comment:
Referring to the discussion in the comments, the following is suggested here, 

```c++
        const std::string msg{"nan or inf value found after UMFPack solve due to singular matrix"};
        OpmLog::debug(msg);
        OPM_THROW_NOLOG(Opm::NumericalIssue, msg);
```

You need to add the following header file
```
#include <opm/common/OpmLog/OpmLog.hpp>
#include <string>
```